setup/MySQL/README.md

```diff
@@ -0,0 +1,9 @@
+## Scripts for MySQL usage
+
+### Usage
+
+```bash
+bash setup/MySQL/convert_mysql.sh <username> <password> <db_name>
+```
+
+If db_name is missing it will use default database name: atpdatabase
```

Comment:
Could you add an example SQL query? A simple question will do. Try "Who is the oldest player to have won a Grand Slam?"